server/test/uk/gov/ons/addressIndex/server/modules/ElasticsearchRepositorySpec2.scala

```diff
@@ -0,0 +1,53 @@
+package uk.gov.ons.addressIndex.server.modules
```

Comment:
The one linked is complete (all tokens are provided), if you want to provide one where not all tokens are provided (I'd say that we can avoid doing it as query modification already requires us to modify corresponding 1 test), you may add them near the linked test in the same manner (the expected and the results are Json object instead of Strings)